WordPress/src/test/java/org/wordpress/android/ui/news/LocalNewsServiceTest.kt

```diff
@@ -37,9 +38,11 @@ class LocalNewsServiceTest {
     }
 
     @Test
-    fun verifyItemEmittedOnGetNewsItem() {
+    fun verifyItemEmittedOnPullInvoked() {
         val observable = localNewsService.newsItemSource()
         observable.observeForever(observer)
-        Mockito.verify(observer).onChanged(newsItem)
+        verify(observer, times(0)).onChanged(any())
```

Comment:
you can do `verify(observer, never())` 😉 or `verifyZeroInteractions(observer)`

Reply:
👍